### ISSUE TITLE
checkboxes for selecting multiple tickets within 1 event, with a maximum of one ticket for each ticket

### ISSUE THREAD

**Issue by minkah1** [eventespresso/event-espresso-core#4171]
At Arnes we organize a free, 3-day conference with many lectures and workshops.

This conference is organized for teachers who receive a certificate of attendance at the end of the conference. To obtain this certificate, they must register through the single sign of one.
At the same time, this conference is also organized for researchers who do not have the option of single sign one, so they are not logged into Wordpress when registering and do not need this type of certificate.

In the current version of Event Espresso, we really miss the feature where there are several different lectures within one event (eg Day 1) and the individual is limited to buying a maximum of one ticket for each lecture. Instead of a drop-down menu of available options, we would prefer to be able to select several different tickets, using checkboxes.

**Comment by garthkoyle** [comment]
> Instead of a drop-down menu of available options, we would prefer to be able to select several different tickets, using checkboxes.

Just to confirm... currently you can do this where the ticket quantity dropdowns has a minimum of 0 and a maximum of 1? That way they can choose any number of lectures, but only one per lecture at that conference?

Are you requesting to change the ticket selection for tickets--with a maximum quantity of one ticket (not one ticket per event)--so that it appears as a checkbox instead of a dropdown?

**Comment by minkah1** [comment]
Yes, with the new update we can have a minimum of 0 and a maximum of 1. 
However, we are still asking if it could appear as a checkbox instead of a dropdown menu.

**Comment by garthkoyle** [comment]
So, in the case that the minimum ticket quantity is 0, and the maximum ticket quantity is 1, we could look into changing the style in the pricing option selector FROM a select box with options of 0 and 1, TO a checkbox.

This would require the following (and possibly more):
- New logic to change the style given the maximum ticket quantity of 1.
- A feature flag for users to control the style, so we don't change the style for people who want or are familiar with the current select box style.
